apitools/base/py/credentials_lib.py

```diff
@@ -384,6 +387,21 @@ def _refresh(self, _):
     self.access_token = token
 
 
+def _GetRunFlowFlags():
+  if FLAGS is None:
```

Comment:
OK. Should I take that approach with this method or `FLAGS is not None` is sufficient?